### PR TITLE
EN_REboot: NGS story quest dialogue intro

### PR DESCRIPTION
**Pull request by allister-translator**
figured i should get this evaluated before i get ahead of myself

### CODE REVIEW

File: Story/msc_010050.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+msc_010050_0010#0,"""DOLLS?! What are they doing over here?!"""
+msc_010050_5010#0,"""Hah!"""
+msc_010050_0020#0,"""Manon!"""
+msc_010050_0030#0,"""“A Pettas Sword, huh... Yeah, I can handle this much!"""
```
Comment (SynthSy) [inline Story/msc_010050.csv]: We're not using global enemy names, so pettas -> pedas

File: Story/msc_010100.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+msc_010100_5010#0,"""Garoa, welcome back. Oh, hey, welcome! Make yourself at home, friend! Meteorn are always welcome around these parts."""
+msc_010100_0010#0,"""Welcome to Aelio Town! On behalf of everyone in the village, welcome."""
```
Comment (SynthSy) [inline Story/msc_010100.csv]: Aelio -> Aerio

File: Story/msc_010130.csv [hunk 3]
```diff
@@ -0,0 +1,48 @@
+msc_010130_0010#0,"""Here, take a seat."""
+msc_010130_0020#0,"""Alright, let's take it from the top."""
+msc_010130_0030#0,"""I am Garoa. I'm the leader of this here village."""
+msc_010130_0040#0,"""That's Aina. She's my daughter."""
+msc_010130_0050#0,"""Hiya!"""
+msc_010130_0060#0,"""And next to her is Manon."""
+msc_010130_0070#0,"""... Hello."""
+msc_010130_0080#0,"""Alright, first, let's talk about where the heck you are."""
+msc_010130_0090#0,"""We call this planet Halpha."""
+msc_010130_0100#0,"""And on it, we're in a village called Aelio Town."""
```
Comment (SynthSy) [inline Story/msc_010130.csv]: Aelio -> Aerio

File: Story/msc_010130.csv [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+msc_010130_0110#0,"""Probably too small to have Town in its name, but hey."""
+msc_010130_0120#0,"""North of here is Central City, the largest of its kind on Halpha."""
+msc_010130_0130#0,"""True to its name, it's pretty much the center of all governing and trade. Your feet will surely carry you there one day."""
+msc_010130_0140#0,"""Now, next topic..."""
+msc_010130_0150#0,"""The DOLLS."""
+msc_010130_0160#0,"""That Pettas Sword you were fighting earlier? It's actually one of the smaller ones."""
```
Comment (SynthSy) [inline Story/msc_010130.csv]: Pettas -> Pedas

File: Story/msc_010130.csv [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+msc_010130_0170#0,"""We Halphians have been fighting DOLLS as long as any of us can remember."""
+msc_010130_0180#0,"""If I had to give a number of years, well, let's see..."""
+msc_010130_0190#0,"""They say it's been for over 500 years."""
+msc_010130_0200#0,"""What she said."""
+msc_010130_0210#0,"""However, there's still a lot that we really don't know about the DOLLS."""
+msc_010130_0220#0,"""Why do they attack us? Where do they even come from? Nobody's got a single clue."""
+msc_010130_0240#0,"""There's one thing everyone knows for sure, though: DOLLS are our enemies, each and every one of 'em."""
+msc_010130_0250#0,"""While we're on the topic of mysteries, there's you Meteorn folks as well."""
+msc_010130_0260#0,"""Meteorn is what we call the folks that crash down from outer space inside one of those pods like the one you were found in."""
```
Comment (SynthSy) [inline Story/msc_010130.csv]: I think here, it should be "Meteorn are" rather than "Meteron is".

File: Story/msc_010140.csv [hunk 6]
```diff
@@ -0,0 +1,29 @@
+msc_010140_0010#0,"""Hey there. It's not much, but I put together a little something for you."""
+msc_010140_0020#0,"""A warm welcome to a newly arrived Meteorn. It's a bit of a Halphian tradition."""
+msc_010140_0030#0,"""Come with me."""
+msc_010142_0040#0,"""It's not everyone, but I had some of the Aelio Town reisdents join us."""
```
Comment (SynthSy) [inline Story/msc_010140.csv]: Aelio -> Aerio

File: Story/msc_010144.csv [hunk 7]
```diff
@@ -0,0 +1,33 @@
+msc_010140_0010#0,"""Hey there. It's not much, but I put together a little something for you."""
+msc_010140_0020#0,"""“A warm welcome to a newly arrived Meteorn. It's a bit of a Halphian tradition."""
+msc_010140_0030#0,"""Come with me."""
+msc_010142_0040#0,"""It's not everyone, but I had some of the Aelio Town reisdents join us."""
```
Comment (SynthSy) [inline Story/msc_010144.csv]: Aelio -> Aerio

File: Story/msc_010144.csv [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+msc_010142_0090#0,"""Cheers!"""
+msc_010142_0092#0,"""Cheers!"""
+msc_010142_0094#0,"""Cheers."""
+msc_010144_0100#0,"""Quite the view, ain't it."""
+msc_010144_0110#0,"""I sure have taken a liking to it. Makes me want to protect this place and everyone in it."""
+msc_010144_0120#0,"""......"""
+msc_010144_0130#0,"""ARKS."""
+msc_010144_0140#0,"""It's the name of the defenders that fight against the DOLLS."""
+msc_010144_0150#0,"""I, as well as Manon, are both members of ARKS. Aina's training to become one."""
+msc_010144_0160#0,"""You looked like you were handling yourself pretty well against that Pettas Sword before I showed up."""
```
Comment (SynthSy) [inline Story/msc_010144.csv]: Pettas -> Pedas

File: Story/msc_010190.csv [hunk 9]
```diff
@@ -0,0 +1,11 @@
+msc_010190_0010#0,"""I can't believe something like this has happened..."""
+msc_010190_0020#0,"""What happened?"""
+msc_010190_0030#0,"""Where's Aina?"""
+msc_010190_0040#0,"""Where's Garoa and the villagers?"""
+msc_010190_0050#0,"""Looks like we got knocked out by that huge DOLLS attack..."""
+msc_010190_0060#0,"""The DOLLS are gone, but... so is all of Aelio Town..."""
```
Comment (SynthSy) [inline Story/msc_010190.csv]: Aelio -> Aerio

File: Story/msc_010220.csv [hunk 10]
```diff
@@ -0,0 +1,8 @@
+msc_010220_0010#0,"""You're coming from Aelio Town, yes?"""
```
Comment (SynthSy) [inline Story/msc_010220.csv]: Aelio -> Aerio

**Review by SynthSy — CHANGES_REQUESTED** [review]

File: Story/msc_011350.csv [hunk 11]
```diff
@@ -0,0 +1,23 @@
+msc_011350_0010#0,"""As you heard, within the Razor Woods,"""
```
Comment (SynthSy) [inline Story/msc_011350.csv]: Resol Forest

File: Story/msc_011350.csv [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+msc_011350_0040#0,"""Yes. THAT Nex Aelio."""
+msc_011350_5010#0,"""Did he just say \""Nex Aelio\""? What's that? \""Nex Aelio\""?"""
+msc_011350_0050#0,"""The last time we encountered Nex Aelio was around 10 years ago."""
+msc_011350_0060#0,"""It wrought the greatest devastation across the entire Aerio region. At least, it was the first such event."""
+msc_011350_0070#0,"""..."""
+msc_011350_0080#0,"""Just recently, Bruder's party was sent to scout the Razor Woods, when suddenly they ceased all transmissions."""
+msc_011350_5020#0,"""What?! Bruder's...?!"""
+msc_011350_0100#0,"""To make matters worse, this was around the time Nex Aelio appeared, followed by increased DOLLS activity in every region."""
+msc_011350_0110#0,"""This means that ARKS will have to be stretched thin in order to deal with every situation."""
+msc_011350_0120#0,"""The mission to send support into Razor Woods, in particular, will be considerably dangerous."""
```
Comment (SynthSy) [inline Story/msc_011350.csv]: Resol Forest

File: Story/msc_011330.csv [hunk 13]
```diff
@@ -0,0 +1,14 @@
+msc_011330_0010#0,"""...Phew."""
+msc_011330_0020#0,"""Nice fighting, Aina."""
+msc_011330_0030#0,"""Thanks, Manon."""
+msc_011330_0040#0,"""We're back online. Daedal signature has been neutralized."""
+msc_011330_0050#0,"""The three of you took down a Daedal? That's amazing!"""
+msc_011330_0060#0,"""What was that?"""
+msc_011330_0070#0,"""Checking now... Wait, what?!"""
+msc_011330_0080#0,"""While patrolling the Razor Woods, Bruder's team went silent..."""
```
Comment (SynthSy) [inline Story/msc_011330.csv]: Resol Forest

**Review by SynthSy — CHANGES_REQUESTED** [review]

File: Story/msc_011350.csv (hunk 11 above)
Comment (allister-translator) [reply Story/msc_011350.csv]: fixed

File: Story/msc_011350.csv (hunk 12 above)
Comment (allister-translator) [reply Story/msc_011350.csv]: fixed

File: Story/msc_011330.csv (hunk 13 above)
Comment (allister-translator) [reply Story/msc_011330.csv]: fixed

**Pull request merged**